client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -149,309 +153,334 @@ export const StockLineForm = ({
         <Grid
           flex={1}
           container
+          flexDirection="column"
           paddingTop={2}
           width="100%"
           flexWrap="nowrap"
           maxWidth={900}
-          gap={isNewModal ? undefined : 10}
         >
-          <Grid
-            container
-            flex={1}
-            flexBasis="50%"
-            flexDirection="column"
-            gap={1}
-          >
-            <StyledInputRow
-              label={t('label.pack-quantity')}
-              Input={
-                <NumericTextInput
-                  autoFocus
-                  disabled={!packEditable}
-                  width={160}
-                  value={
-                    draft.totalNumberOfPacks ? draft.totalNumberOfPacks : 0
-                  }
-                  onChange={totalNumberOfPacks =>
-                    onUpdate({ totalNumberOfPacks })
-                  }
-                  {...getDosesProps(draft.totalNumberOfPacks)}
-                />
-              }
-            />
-            {!packEditable && (
-              <>
+          {!isNewModal && (
+            <Box paddingBottom={1}>
+              <Box display="flex">
+                <Box style={{ textAlign: 'end', whiteSpace: 'nowrap' }}>
+                  <FormLabel
+                    sx={{
+                      fontWeight: 'bold',
+                      display: 'inline-block',
+                      width: '100px',
+                    }}
+                  >
+                    {t('label.item')}:
+                  </FormLabel>
+                </Box>
+                <Box paddingLeft={1} paddingRight={1.5}>
+                  <Box>
+                    <Link
+                      to={RouteBuilder.create(AppRoute.Catalogue)
+                        .addPart(AppRoute.Items)
+                        .addPart(draft.itemId)
+                        .build()}
+                    >
+                      {draft.item.name}
+                    </Link>
+                  </Box>
+                </Box>
+              </Box>
+            </Box>
+          )}
+          <Box>
```

Comment:
This looks like a lot of extra boxes... I don't think they're all needed 👀 

I tried like this and it looks the same - not a biggie, but a good practice to add as few nodes as possible 😁 

```suggestion
          {!isNewModal && (
            <Box display="flex" marginBottom={2}>
              <FormLabel
                sx={{
                  fontWeight: 'bold',
                  display: 'inline-block',
                  width: '100px',
                  textAlign: 'end',
                  whiteSpace: 'nowrap',
                }}
              >
                {t('label.item')}:
              </FormLabel>
              <Box paddingLeft={1} paddingRight={1.5}>
                <Link
                  to={RouteBuilder.create(AppRoute.Catalogue)
                    .addPart(AppRoute.Items)
                    .addPart(draft.itemId)
                    .build()}
                >
                  {draft.item.name}
                </Link>
              </Box>
            </Box>
          )}
```